ckanext/pages/theme/templates_main/ckanext_pages/blog_list.html

```diff
@@ -2,7 +2,7 @@
 
 {% block page_primary_action %}
   {% if h.check_access('ckanext_pages_update', {}) %}
-    {% link_for _('Add Article'), controller='ckanext.pages.controller:PagesController', action='blog_edit', page='', class_='btn btn-primary', icon='plus-sign-alt' %}
+    {% link_for _('Add Article'), controller='ckanext.pages.controller:PagesController', action='blog_edit', page='', class_='btn btn-primary', icon='plus-square' %}
```

Comment:
I can see how the changes of the classes in the HTML are backwards compatible (since they only add classes), but I'm not so sure about these changes here which replace the old icon name in the helper call. Wouldn't this break things on old CKAN versions?